go/grpcwebproxy/main.go

```diff
@@ -34,6 +34,7 @@ var (
 
 	flagAllowAllOrigins = pflag.Bool("allow_all_origins", false, "allow requests from any origin.")
 	flagAllowedOrigins  = pflag.StringSlice("allowed_origins", nil, "comma-separated list of origin URLs which are allowed to make cross-origin requests.")
+	flagAllowedHeaders  = pflag.StringSlice("allowed_headers", []string{}, "comma-separated list of headers which are allowed to propagate to the gRPC backend.")
 
 	runHttpServer = pflag.Bool("run_http_server", true, "whether to run HTTP server")
 	runTlsServer  = pflag.Bool("run_tls_server", true, "whether to run TLS server")
```

Comment:
This is from my previous PR that was already merged. I am not sure what would be right thing to do. Should I rebase my commits into ws-request-headers (the branch already merged) and make PR from there?

Reply:
Yes, rebase on current master please.

Reply:
Well, yet another PR: #589